pkg/kudoctl/cmd/get/get.go

```diff
@@ -0,0 +1,98 @@
+package get
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+	"github.com/pkg/errors"
+
+	"github.com/xlab/treeprint"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/check"
+)
+
+// Options defines configuration options for the install command
+type Options struct {
```

Comment:
is this copy-paste error? How is skipInstance relevant to get instances? 🤔

Reply:
I wouldn't call it that. More good intensions to make parts of the CLI following same patterns/look alike.